### ISSUE TITLE
Alt+Tab breaks mod

### ISSUE THREAD

**Issue by decoytrix** [yamashi/CyberEngineTweaks#519]
If I Alt Tab to other windows and come back to the game, my keybind will not longer pull up the console. Confirmed issue whether playing in fullscreen or windows mode. Only resolution is to restart the game any time I'm planning on using my PC for anything else. 

(Also, the Discord support link from the Nexus page is invalid)

**Comment by WSSDude420** [comment]
Dont know what your bind is but i suspect it is bound to either Alt or Tab?

Does this happen on newest version as well?

**Comment by decoytrix** [comment]
Initially I thought that might be it as opening steam overlay before alt+tabbing would keep the tweak overlay from breaking, but when I launch through Vortex's shortcut (which gets no steam overlay) I'll have the same issue.
I figured out it isn't going to another window that breaks it, but SPECIFICALLY alt+tab.  If I hit the Tweaks overlay key and then just drag my mouse to the other screen and another program, I can switch back to Cyberpunk with no issue. I can also Alt+tab back to that window as long as I'm not IN Cyberpunk when I use Alt+Tab. 

Happily I've found enough work-a-rounds that this isn't game breaking or anything, but if the info helps others or anything, that's what I've figured out so far.

**Comment by WSSDude420** [comment]
One more thing I remember is that mods can actually cause this in some very rare instances, although this should be very slim chance nowadays with all the guards in place,

There was a time when something like Game.GetPlayer() called each onUpdate call caused problems with alt tabs.
Maybe this is something in similar vain (above is not an issue anymore I believe, but could be something similar is happening now with something else).

If you run vanilla CET, does it still crash?
This is where I would start.

We do not give any guarantees about mods and we do not do support for them :P
If issue can be proven to be caused by clean CET installation, good. But if not, we cannot help much.

To recap, as mentioned, this could be issue with some other overlay or monitoring app OR by some mod.
If this happens with vanilla CET when nothing else is there to interfere, we can try to look for something but this is where I would start this off :)
